src/views/Appointments/Booking.vue

```diff
@@ -0,0 +1,193 @@
+<!--
+  - @copyright 2021 Christoph Wurst <christoph@winzerhof-wurst.at>
+  -
+  - @author 2021 Christoph Wurst <christoph@winzerhof-wurst.at>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<div class="booking">
+		<div class="booking__config-user-info">
+			<Avatar
+				:user="userInfo.uid"
+				:display-name="userInfo.displayName"
+				:disable-tooltip="true"
+				:disable-menu="true"
+				:size="64" />
+			<div class="booking__display-name">
+				<strong>{{ userInfo.displayName }}</strong>
+			</div>
+			<h2 class="booking__name">
+				{{ config.name }}
+			</h2>
+			<span class="booking__description">{{ config.description }}</span>
+		</div>
+		<div class="booking__date-selection">
+			<h3>{{ $t('calendar', 'Select date') }}</h3>
+			<div class="booking__date">
+				<DatetimePicker v-model="selectedDate"
+					type="date"
+					@change="fetchSlots" />
+			</div>
+			<div class="booking__time-zone">
+				<TimezonePicker v-model="timeZone" @change="fetchSlots" />
+			</div>
+		</div>
+		<div class="booking__slot-selection">
+			<h3>{{ $t('calendar', 'Select slot') }}</h3>
+
+			<div class="booking__slots">
+				<AppointmentSlot v-for="slot in slots"
+					:key="slot.start"
+					:start="slot.start"
+					:end="slot.end"
+					:time-zone-id="timeZone" />
+			</div>
+			<button class="button primary"
+				@click.prevent.stop="showAppointmentDetails ">
+				{{ $t('calendar', 'Next') }}
```

Comment:
what's the purpose of this button @GretaD?

Reply:
to show the modal, or the next step of booking the appointment, i was not sure how to do that. So i did the easiest think that i could think of, but we can remove it/or do smth else.